src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/PlacementRequestsTest.kt

```diff
@@ -1620,7 +1620,7 @@ class PlacementRequestsTest : IntegrationTestBase() {
     }
 
     @Test
-    fun `Withdraw Placement Request returns 200, sets isWithdrawn to true and sends email to CRU when no booking`() {
+    fun `Withdraw Placement Request returns 200, sets isWithdrawn to true and sends email to CRU when no booking and applicant`() {
```

Comment:
should we try to be a bit more explicit about the scenario here, e.g. "...when their is no booking and an applicant exists as the PR was automatically created on submission of an application with known release date". A bit verbose but might be helpful documentation?

Reply:
Good idea. I've had a go at rewording. I'm hoping over the next few weeks we can keep refactoring to the point where this logic makes more sense and doesn't necessarily need so much explanation (but at the moment I completely agree it does!)